p2p/src/main/java/bisq/network/p2p/network/TorNetworkNode.java

```diff
@@ -187,6 +187,7 @@ private BooleanProperty torNetworkNodeShutDown() {
                 long ts = System.currentTimeMillis();
                 log.debug("Shutdown torNetworkNode");
                 try {
+                    if (tor == null) tor = Tor.getDefault();
```

Comment:
NIT: I think we decided at some point not to use conditionals within one line anymore